### PR TITLE
tests: ui/inline-consts: add issue number to a test, rename other tests

### PR DESCRIPTION
**Pull request by matthiaskrgr**
rename other tests from a_b_c to a-b-c

### CODE REVIEW

File: tests/ui/inline-const/cross-const-control-flow-125918.rs
Comment (workingjubilee) [inline tests/ui/inline-const/cross-const-control-flow-125918.rs:1]: The issue is actually https://github.com/rust-lang/rust/issues/125846
This number is the PR, https://github.com/rust-lang/rust/pull/125918

Can we add them both as links to the initial comments of this file? Perhaps a remark about how this is to catch a regression introduced by treating inline const blocks as the same as other anon consts?

**Comment by jieyouxu** [comment]
@matthiaskrgr just curious, is there a specific reason for renaming the tests use `-` over `_`?

**Comment by matthiaskrgr** [comment]
I just looked at what the majority of the other tests were doing :shrug:

**Review by davidtwco — APPROVED** [review]

**Pull request merged**